nixos/modules/services/torrent/flood.nix

```diff
@@ -0,0 +1,115 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.flood;
+in
+{
+  meta.maintainers = with lib.maintainers; [ thiagokokada ];
+
+  options.services.flood = {
+    enable = lib.mkEnableOption "flood";
+    package = lib.mkPackageOption pkgs "flood" { };
+    openFirewall = lib.mkEnableOption "" // {
```

Comment:
What does this line do?

Reply:
It opens the firewall ports: https://github.com/NixOS/nixpkgs/pull/317530/files#diff-9cac63135951b34a3d402ec244c73ab992337c7eec80645de5a755b982c25371R110-R112